esp32-wroom-rp/src/spi.rs

```diff
@@ -317,6 +363,34 @@ where
         Ok(response_param_buffer)
     }
 
+    fn read_response16(&mut self) -> Result<NinaResponseBuffer, Error> {
+        // let response_length_in_bytes = self.get_byte().ok().unwrap() as usize;
+
+        let number_of_params = self.get_byte().unwrap() as usize;
+
+        // if response_length_in_bytes > MAX_NINA_PARAMS {
+        //     return Err(ProtocolError::TooManyParameters.into());
+        // }
+
+        let mut response_param_buffer: NinaResponseBuffer = [0; MAX_NINA_RESPONSE_LENGTH];
```

Comment:
And then per my comment above about a generic NinaResponseBuffer trait/type, this could instantiate a `LargeNinaResponseBuffer`.